log.js

```diff
@@ -4,9 +4,9 @@ var path = require('path')
 
 module.exports = function (dir, config) {
   config = config || {}
-    
+
   var log = OffsetLog(
-    path.join(dir, 'log.bipf'),
+    path.join(dir, 'flumelog.bipf'),
```

Comment:
Guess this sort of depends on #19. With #19 this shouldn't be needed?

Reply:
Yes, I rebased this PR now with the latest changes from master